pysph/sph/wc/transport_velocity.py

```diff
@@ -734,3 +734,72 @@ def post_loop(self, d_idx, d_wij, d_p, d_rho):
 
         # update the density from the pressure Eq. (28)
         d_rho[d_idx] = self.rho0 * (d_p[d_idx]/self.p0 + self.b)
+
+
+class NoSlipVelocityExtrapolation(Equation):
+    '''No Slip boundary condition on the wall
```

Comment:
Is this really no-slip or is it a slip-wall?

Reply:
Doesn't it depend on whether viscosity is used between this wall and the fluid?

Reply:
It is slip wall.

Yes, but we cannot just disbale viscosity. the wall velocity needs to be same as fluid so that not gradient develops.